topics/client.md

```diff
@@ -1,105 +1,204 @@
 [//]: # (title: Client Overview)
```

Comment:
All samples are using code directly. They should be using code snippet references.

Reply:
done